Mage/src/main/java/mage/game/combat/Combat.java

```diff
@@ -1229,7 +1229,11 @@ public boolean checkBlockRestrictionsAfter(Player player, Player controller, Gam
                         if (!effect.canBlockCheckAfter(ability, game, true)) {
                             if (controller.isHuman()) {
                                 controller.resetPlayerPassedActions();
-                                game.informPlayer(controller, blockingCreature.getLogName() + " can't block this way.");
+                                
+                                // Both write a line in the logs, and popup a message to the blocking player, for full clarity.
+                                String message = " can't block this way.";
+                                game.informPlayer(controller, blockingCreature.getIdName() + message);
+                                game.informPlayers(blockingCreature.getLogName() + message);
```

Comment:
Wrong user's action must be hidden from other players (it could reveal a player's strategy). No needs in `informPlayers` logs here and other new places

Also I recommends to add source of the restriction by `+ CardUtil.getSourceLogName(game, ability)`